File: Python/1365-how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        count = [0] * 101
        for i in nums:
            count[i] += 1
        previous = 0
        for i, c in enumerate(count):
            if c !=0:
                count[i] =  previous
                previous += c
        return [count[i] for i in nums]

File: C++/1365-how-many-numbers-are-smaller-than-the-current-number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
      vector<int> ans;
      int count[101] = {0};
      for(auto& i:nums){
        count[i]++;
      }
      for(int i =1; i<=100; i++){
        count[i] += count[i-1];
      }
      for(int i = 0; i < nums.size(); i++){
        if(nums[i]==0)
          ans.push_back(0);
        else
          ans.push_back(count[nums[i] - 1]);
      }
      return ans;
    }
};
